ff/serving/coordinator/coordinator.go

```diff
@@ -85,40 +108,37 @@ func NewCoordinator(meta *metadata.Client, logger *zap.SugaredLogger, cli *clien
 		EtcdClient: cli,
 		KVClient:   &kvc,
 		Spawner:    spawner,
+		Timeout:    5,
 	}, nil
 }
 
 const MAX_ATTEMPTS = 1
 
 func (c *Coordinator) WatchForNewJobs() error {
+	c.Logger.Debug("Watching for new jobs")
 	getResp, err := (*c.KVClient).Get(context.Background(), "JOB_", clientv3.WithPrefix())
 	if err != nil {
 		return err
 	}
 	for _, kv := range getResp.Kvs {
-		fmt.Println("FOUND:", kv)
-		go func() {
+		go func(kv *mvccpb.KeyValue) {
 			err := c.executeJob(string(kv.Key))
 			if err != nil {
-				fmt.Println(err)
+				c.Logger.Debugw("Error executing job", "error", err)
 			}
-		}()
+		}(kv)
 	}
 	for {
-		fmt.Println("WATCHING")
 		rch := c.EtcdClient.Watch(context.Background(), "JOB_", clientv3.WithPrefix())
 		for wresp := range rch {
-			fmt.Println("FOUND:", wresp)
 			for _, ev := range wresp.Events {
-				fmt.Println("FOUND:", ev)
 				if ev.Type == 0 {
-					fmt.Println("EXECUTING:", ev.Kv.Key)
-					go func() {
+					go func(ev *clientv3.Event) {
 						err := c.executeJob(string(ev.Kv.Key))
 						if err != nil {
-							fmt.Println(err)
+							c.Logger.Debugw("Error executing job", "error", err)
```

Comment:
Change these so we can tell them apart if an error occurs. 
Can be something like "Error executing job: Initial Search"/ "Error executing job: Polling Search"